app/packages/operators/src/loader.tsx

```diff
@@ -4,21 +4,15 @@ import { useEffect, useState } from "react";
 import { useRecoilValue, useSetRecoilState } from "recoil";
 import { registerBuiltInOperators } from "./built-in-operators";
 import { useOperatorPlacementsResolver } from "./hooks";
-import { executeStartupOperators, loadOperatorsFromServer } from "./operators";
+import { loadOperatorsFromServer } from "./operators";
 import {
   availableOperatorsRefreshCount,
   operatorsInitializedAtom,
 } from "./state";
 
-let startupOperatorsExecuted = false;
-
 async function loadOperators(datasetName: string) {
   registerBuiltInOperators();
   await loadOperatorsFromServer(datasetName);
-  if (!startupOperatorsExecuted) {
-    executeStartupOperators();
-    startupOperatorsExecuted = true;
-  }
```

Comment:
I believe the doc string below is incorrect now? `OperatorCore` now queues start-up operators?

Reply:
Yah, good point. I will update the doc string below.